crates/bevy_egui/src/systems.rs

```diff
@@ -0,0 +1,132 @@
+use std::sync::Arc;
+
+use bevy::{prelude::*, render::{draw::DrawContext, render_graph::RenderGraph, renderer::RenderResourceBindings}};
+
+use egui::{Rgba, Slider, containers::Window};
+
+use crate::{render::AddEguiSystemNode, components::EguiJobsDescriptor, egui_node::EguiNode, egui_node::{EguiSystemNode, FAKE_RAW_INPUT}, egui_ui::EguiUi};
+
+pub fn egui_test_system(
+    mut egui: ResMut<EguiUi>,
+) {
+    let ui = match egui.ui.as_mut() {
+        None => return,
+        Some(ui) => ui
+    };
+
+    let mut value = 1.0;
+
+    println!("Rendering ui stuffs!");
+
+    Window::new("Debug").fixed_pos([500.0, 200.0]).show(ui.ctx(), |ui| {
+        ui.label(format!("Hello, world {}", 123));
+        ui.add(egui::widgets::Label::new("Hello world!").heading());
+        if ui.add(egui::widgets::Button::new("Save").fill(Some(Rgba::RED.into()))).clicked {
+            println!("Save clicked!");
+        }
+        ui.add(Slider::f32(&mut value, 0.0..=1.0).text("float"));
+    });
+}
```

Comment:
If this is the kind of system users of the egui integration would use to define a new UI I'd probably move it out to your demo binary main.rs, instead of building it into this crate.